### ISSUE TITLE
Error, Notification: Screen reader announces incorrect expand/collapse state for 'Console' button.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1054]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1245645

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Create 'Sample Database'. 
Turn on JAWS and navigate to Expand/Collapsed button in the bottom and expand and collapse it.
Observe the screen reader behavior.
Verify the issue.

Actual Result:
Screen reader does not announces correct expand/collapse state for 'Console' button. It is reading as 'collapsed' on 'expanding' and as 'expanded' on 'collapsing' it.

Expected Result:
Screen reader should announce correct expand/collapse state for 'Console' button. It should read as 'expanded' on expanding and 'collapsed' on collapsing.

https://user-images.githubusercontent.com/8519407/132715931-a81afd31-e608-470b-a9df-49f901f728a8.mp4
